lib/shell/latex_run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# This script is copied into tmp/rails-latex/$WORK_DIR/ and executed by rails-latex
+
+WORK_DIR=$(basename "$PWD")
+docker exec -t $LATEX_CONTAINER_NAME $LATEX_BUILD_PATH $WORK_DIR
```

Comment:
We should check that these environment variables exist - either here or in the application.rb file. Then report an error so that the config can be checked.
$LATEX_CONTAINER_NAME 
$LATEX_BUILD_PATH

Reply:
Should we default LATEX_BUILD_PATH to /texlive/shell/latex_build.sh if it is not set?